### DISCUSSION TITLE
500 Server Error when loading site

### DISCUSSION THREAD

**Discussion by JRJurman** [adamghill/unsuckjs.com#15] [General]
Looks like the site is throwing a 500 😞 
<img width="1904" alt="image" src="https://github.com/adamghill/unsuckjs.com/assets/326557/893a12ae-ec03-4f35-8376-6331a72d9997">

**Comment by adamghill** [comment]
Yikes, thanks! Deploying a fix now.

**Reply by adamghill** [reply]
The site should be back online! Thanks again.
